build/common.sh

```diff
@@ -378,13 +378,9 @@ function kube::build::short_hash() {
 # a workaround for bug https://github.com/docker/docker/issues/3968.
 function kube::build::destroy_container() {
   "${DOCKER[@]}" kill "$1" >/dev/null 2>&1 || true
-  if [[ $("${DOCKER[@]}" version --format '{{.Server.Version}}') = 17.06.0* ]]; then
-    # Workaround https://github.com/moby/moby/issues/33948.
-    # TODO: remove when 17.06.0 is not relevant anymore
```

Comment:
i don't think 17.06 is out of support yet in k8s.

Reply:
Yeah, I don't think so either.

Reply:
+1 - `kubeadm` still supports Docker 17.06.0 [here](https://github.com/kubernetes/kubernetes/blob/59a86822cb966ffb77a702321c89bb4e6f1c1c89/cmd/kubeadm/app/util/system/docker_validator_test.go#L65).

Thank you for having an active eye towards fixing todos. Maybe we can close this and reopen it in a couple of months/year when 17.06.0 support is dropped.

Reply:
Thanks, I'll close the PR.